scss/_variables.scss

```diff
@@ -318,9 +318,10 @@ $container-max-widths: (
 // Set the number of columns and specify the width of the gutters.
 
 $grid-columns:                12 !default;
-$grid-gutter-width:           30px !default;
+$grid-gutter-width:           1.5rem !default;
```

Comment:
@mdo  I prefer 2rem close to 30px. Please double check that you intend that this 1.5rem affects the container padding. You can see the examples here:

* Before: https://twbs-bootstrap.netlify.com/docs/4.3/examples/navbars/
* After: https://deploy-preview-28517--twbs-bootstrap.netlify.com/docs/4.3/examples/navbars/

Reply:
The container padding can work independent from the gutter width. I'm ok with changing the container padding to `1.5rem`.

Reply:
Ok, I've now changed it to `1rem`, which looks more or less the same as it used to look like.